drivers/flash/spi_nor.c

```diff
@@ -1106,6 +1210,13 @@ static int spi_nor_configure(const struct device *dev)
 #endif /* CONFIG_FLASH_PAGE_LAYOUT */
 #endif /* CONFIG_SPI_NOR_SFDP_MINIMAL */
 
+#if DT_INST_NODE_HAS_PROP(0, mxicy_high_perf_mode)
```

Comment:
This is supposed to be executed always when the new binding is used, so it can also be used to disable high performance mode through a firmware update (only soft-boot). But I see that I should have used the same condition as around the functions declarations: `#if DT_HAS_COMPAT_STATUS_OKAY(mxicy_mx25r_spi)`. Will update.